intera_examples/scripts/camera_display.py

```diff
@@ -84,12 +84,7 @@ def main():
 
     print("Initializing node... ")
     rospy.init_node('camera_display', anonymous=True)
-    try:
-        cameras = intera_interface.Cameras()
-    except OSError as e:
-        rospy.logfatal("Could not find all of the expected cameras for this robot.\n"
-                "Please contact Rethink support: support@rethinkrobotics.com")
-        return
+    cameras = intera_interface.Cameras()
     if not cameras.verify_camera_exists(args.camera):
         rospy.logerr("Invalid camera name, exiting the example.")
```

Comment:
This pull request actually changes the message here. Something like the following would make more sense:
```
     if not cameras.verify_camera_exists(args.camera):
          rospy.logerr("Could not detect the specified camera, exiting the example.")
```

Reply:
Sure